server/Config.ts

```diff
@@ -24,9 +24,10 @@ export class Config {
   public readonly configOptions: ConfigOptions = Config.readConfigFromFile();
   public readonly sessionSecret = process.env.SESSION_SECRET || uuid();
   public readonly cookieMaxAgeMs = parseInt(process.env.COOKIE_MAX_AGE_MS || `${DAYS_30_MS}`, 10);
-  public readonly disableAuthOnApi = !!process.env.DISABLE_AUTH_ON_API;
+  public readonly disableAuth = !!process.env.DISABLE_AUTH;
```

Comment:
Couldn't we just infer this by not having any other authentication method available? So like configuring no slack authentication (for now) disables all authentication completely.

Reply:
Good idea!
However, we currently enforce the slack configuration, so changing this is not completely trivial.
So I would extract this into a separate issue?

Reply:
Ok for me.